Samples/XboxWdpDriver/Operations/AppOperation.cs

```diff
@@ -70,6 +77,42 @@ public static void HandleOperation(DevicePortal portal, ParameterHelper paramete
                     packagesTask.Wait();
                     Console.WriteLine(packagesTask.Result);
                 }
+                else if (operationType.Equals("listrunningapps"))
+                {
+                    Task<List<string>> packagesTask = portal.GetRunningAppsAsync();
```

Comment:
Nit: the branches for getting running and suspended apps are pretty similar.  Not sure if its worth writing some kind of wrapper function to re-use the logic.  For example a function calls could look something like this:
printAppList(packagesTask, "Running") 
printAppList(packagesTask, "Suspended")